packages/terra-site/src/examples/form/examples/Input.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Input from 'terra-form/lib/components/Input';
+
+const InputExamples = () => (
+  <form>
+    <Input
+      name="foo"
+      defaultValue="bar"
+    />
+  </form>
+);
+
+export default InputExamples;
```

Comment:
I decided to add it for this pull request. Setups the easy addition of future controlled inputs.

https://github.com/cerner/terra-core/pull/352/commits/33ffe7b0aba15bcf8a600b929fea2d7f7d0c3e30